ceph_installer/util.py

```diff
@@ -163,7 +163,13 @@ def make_setup_script(url):
 fi
 
 echo "--> creating new user with disabled password: ansible"
-adduser --disabled-password --gecos "" ansible
+if test -f /etc/redhat-release ; then
+    # flags are not compatible in Ubuntu vs. RHEL
+    adduser ansible
```

Comment:
You'd still need to disable the password for the ``ansible`` user on Ubuntu, right?